.github/workflows/test.yml

```diff
@@ -1,29 +1,45 @@
 name: Test
 
 on:
-  - pull_request
+  pull_request: {}
+  push:
+    branches:
+      - master
+
+env:
+  BUNDLE_WITHOUT: release
 
 jobs:
-  test:
+  spec:
     runs-on: ubuntu-latest
     strategy:
-      fail-fast: false
+      fail-fast: true
```

Comment:
Do we really want to fail fast? I generally prefer to know if it's a specific Ruby version. The tests are also really fast here, so we're not saving that much.

Reply:
I am fine with false

Reply:
done